layouts/authors/list.html

```diff
@@ -0,0 +1,35 @@
+{{ define "main" }}
+{{ $member := (index $.Site.Data.team .Params.slug ) }}
+<div class="grid bg-blue"><div class="content-wrap">
+<section id="main" class="author">
+  <div class="flex">
+  <div style="flex: 1; margin-left: 1em">
+    <h1>{{ $member.fullName }}</h1>
+    <div class="links">
+    	{{ if $member.github }}<a href="https://github.com/{{ $member.github }}" target="_blank"><i class="fab fa-github"></i></a>{{ end }}
+    	{{ if $member.twitter }}<a href="htts://twitter/{{ $member.twitter }}" target="_blank"><i class="fab fa-twitter"></i></a>{{end}}
```

Comment:
```suggestion
    	{{ if $member.twitter }}<a href="https://twitter/{{ $member.twitter }}" target="_blank"><i class="fab fa-twitter"></i></a>{{end}}
```

Skontah dok sam cackao sajt da fali `https` za twitter